### PR TITLE
adding empty input checking and removing default input outline

### LINKED ISSUE #10: no input checking
empty Input is causing an error.
do you want me to fix this?

### PR DESCRIPTION
**Pull request by firasmlt**
Resolves #10
adding empty input checking
also, I removed the default text input outline to improve the styling

### CODE REVIEW

**Comment by firasmlt** [comment]
you can take a look at the implementation [here](https://www.loom.com/share/7ec046cd0c0842b0ba3df66339c791fd).

File: pages/index.js [hunk 1]
```diff
@@ -32,11 +38,12 @@ export default function Home() {
         <h3>Name my pet</h3>
         <form onSubmit={onSubmit}>
           <input
+            className={!validInput && styles.invalidInput}
             type="text"
             name="animal"
             placeholder="Enter an animal"
             value={animalInput}
-            onChange={(e) => setAnimalInput(e.target.value)}
+            onChange={onChangeHandler}
```
Comment (narinluangrath) [inline pages/index.js:46]: Have you considered using the [`minlength` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/minlength)?

Comment (firasmlt) [reply pages/index.js:46]: I've added it now, can you take a look?

Comment (narinluangrath) [reply pages/index.js:46]: LGTM

**Comment by schnerd** [comment]
Thanks for opening this and sorry it took us a while to review. While doing client-side validation here can be helpful, I think addressing this issue with server-side validation (see 259a13a2c04b6ffff33da11ba57cdc9d333851e5) is slightly more robust.

Your changes looks good here, but our goal for the quickstart is to keep the code very simple, and having both client-side and server-side validation is probably unnecessary, so I'll close this PR. Appreciate you reporting the original issue!

**Pull request closed without merging**